pkgs/by-name/te/teams-for-linux/package.nix

```diff
@@ -0,0 +1,128 @@
+{
+  lib,
+  stdenvNoCC,
```

Comment:
No reason to use this over
```suggestion
  stdenv,
```

Reply:
You're right, looks like buildNpmPackage already uses stdenv.